cli/azd/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
+## 1.4.0 (2023-10-05)
```

Comment:
We should add an entry about the remote environment stuff, now that @wbreza's changes have been landed.